packages/ckeditor5-engine/src/conversion/upcasthelpers.js

```diff
@@ -653,6 +670,14 @@ function upcastAttributeToMarker( config ) {
 	return ( evt, data, conversionApi ) => {
 		const attrName = `data-${ config.view }`;
 
+		// This converter wants to add a model element, marking a marker, before/after an element (or maybe even group of elements).
+		// To do that, we can use `data.modelRange` which is set on an element (or a group of elements) that has been upcasted.
+		// But, if the processed view element has not been upcasted yet (it does not have a converted), we need to
+		// fire conversion for its children first, then we will have `data.modelRange` available.
+		if ( !data.modelRange ) {
+			data = Object.assign( data, conversionApi.convertChildren( data.viewItem, data.modelCursor ) );
+		}
```

Comment:
This is the actual "fix" part.

Reply:
After adding this part, though, test `can be overwritten using converterPriority` started to fail because `high` priority was used and this converter was fired before paragraph converter.